app/src/main/java/org/simple/clinic/contactpatient/ContactPatientBottomSheet.kt

```diff
@@ -316,11 +316,11 @@ class ContactPatientBottomSheet : BaseBottomSheet<
     )
```

Comment:
Multiple things are happening in one commit, can we rename and remove in two different commits

Reply:
It's mostly just reshuffling of functions, extracting them out. I can separate the renaming functions to a new commit, though.

Reply:
done ✅